### PR TITLE
PUA / UI part time

### LINKED ISSUE #374: Content updates

### PR DESCRIPTION
**Pull request by sharonwarner**
uiPartTime
pua

===

Resolves #374 

- [x] Snapshots updated, if output HTML/JS has changed (`npm run test:update-snapshots`)
- [ ] Changes reviewed on mobile using devtools, if output HTML/CSS has changed
- [x] Spanish checked by adding `?lng=es` to URL e.g. `/guide/benefits?lng=es`, if Spanish updated
- [ ] Automated tests added, if new functionality added
- [ ] Documentation (e.g. READMEs) updated, if relevant
- [ ] IE11 and Edge compatibility confirmed via manual testing in Browserstack, if new libraries added or newish JS/HTML/CSS features used for the first time in this codebase
- [ ] Accessibility & performance audit run using Google Lighthouse, if major changes made

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -263,15 +263,15 @@
         "tooSickNumberOfDays": "If yes, enter the number of days (1-7) you were unable to work.",
         "help-tooSickNumberOfDays": "You must report the number of days that work was missed.",
         "couldNotAcceptWork": "Was there any reason (other than sickness, injury or the disaster) that you could not have accepted full-time work, each workday?",
-        "help-couldNotAcceptWork": "Other than being sick, injured or disaster, could full-time work have been accepted?",
-        "didYouLook": "Did you look for work or contact your last employer, or, if self employed, did you attempt to resume self- employment?",
-        "help-didYouLook": "The work search requirements on your notice of Unemployment Insurance Award must be followed. Work Searches may include in-person, mail, telephone, or Internet contacts with employers. Was there any attempt to look for work or resume work?",
+        "help-couldNotAcceptWork": "Other than being sick, injured, or the disaster, could full-time work have been accepted? ",
```
Comment (kalvinwang) [inline public/locales/en/translation.json:266]: is the extra space at the end intentional? just checking

Comment (sharonwarner) [reply public/locales/en/translation.json:266]: No! Thank you!

File: public/locales/en/translation.json [hunk 2]
```diff
@@ -263,15 +263,15 @@
         "tooSickNumberOfDays": "If yes, enter the number of days (1-7) you were unable to work.",
         "help-tooSickNumberOfDays": "You must report the number of days that work was missed.",
         "couldNotAcceptWork": "Was there any reason (other than sickness, injury or the disaster) that you could not have accepted full-time work, each workday?",
```
Comment (nguyen-tina) [inline public/locales/en/translation.json:265]: "each workday" should be bold

Comment (sharonwarner) [reply public/locales/en/translation.json]: updated, and uploaded new screenshot!

**Pull request merged**